go/consensus/tendermint/apps/registry/genesis.go

```diff
@@ -75,6 +75,25 @@ func (app *registryApplication) InitChain(ctx *abci.Context, request types.Reque
 			}
 		}
 	}
+	for _, v := range st.SuspendedRuntimes {
+		app.logger.Debug("InitChain: Registering genesis suspended runtime",
+			"runtime_owner", v.Signature.PublicKey,
+		)
+		if err := app.registerRuntime(ctx, state, v); err != nil {
+			app.logger.Error("InitChain: failed to register runtime",
+				"err", err,
+				"runtime", v,
+			)
+			return errors.Wrap(err, "registry: genesis suspended runtime registration failure")
+		}
+		var rt registry.Runtime
```

Comment:
Instead of doing this, just make `app.registerRuntime` return `(*registry.Runtime, error)`?

Otherwise, the `cbor.Unmarshal` call failing is a sign that something when absolutely horrifically wrong since the runtime registration does an Unmarshal.